index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please remember for future cases that your commit messages should start with a capital letters. They will be more consistent and it will look more professional